src/__integrationtests__/utils.ts

```diff
@@ -9,9 +9,6 @@ import CType from '../ctype/CType'
 import { getOwner } from '../ctype/CType.chain'
 import Identity from '../identity/Identity'
 
-// FIXME: check with weights
-// export const GAS = new BN(1_000_000)
-export const GAS = new BN(125_000_000)
```

Comment:
With the new Substrate two update, we may be able to handle the weights better. 

https://github.com/paritytech/substrate/pull/6789

At the moment, I am not sure what will do about the whole weights and transaction costs on the demo-client.

Reply:
Okay lets remove this for now, and maybe read later when we can play with the weights.